### DISCUSSION TITLE
Nextcloud AIO fails to start after updating to 7.9.1 - Fulltextsearch issue

### DISCUSSION THREAD

**Discussion by vawaver** [nextcloud/all-in-one#4026] [Questions] [answered]
After updating to Nextcloud AIO 7.9.1, the application fails to start. I have traced the issue to the Fulltextsearch docker container. I am attaching the logs from both docker containers.

I have also tried deleting all containers and starting over without existing docker containers. This did not help.

**Additional information:**

-    I am running Nextcloud AIO on an 8-core CPU and 12 GB RAM.
-    Operating system - Ubuntu server 22.04 - latest updates

Thank you for your time and consideration.
![portainer](https://github.com/nextcloud/all-in-one/assets/52236863/f98eead3-04e5-45d1-9f91-5233c1e13bd7)
![fulltextsearch](https://github.com/nextcloud/all-in-one/assets/52236863/5d79772e-7906-4cd4-ad74-b918dc12d7fd)
![nextcloud-docker](https://github.com/nextcloud/all-in-one/assets/52236863/f77926d2-72cb-4003-b3b3-3a658c2e4d24)
[_nextcloud-aio-fulltextsearch_logs.txt](https://github.com/nextcloud/all-in-one/files/13885946/_nextcloud-aio-fulltextsearch_logs.txt)
[_nextcloud-aio-nextcloud_logs.txt](https://github.com/nextcloud/all-in-one/files/13885947/_nextcloud-aio-nextcloud_logs.txt)

**Comment by szaimen** [comment, marked as answer]
Hi, I tried yesterday and unfortunately cannot reproduce the problem so not sure how to debug this further or fix this as tiny ins included in the image, so it seems like the image is faulty in your case...

**Comment by vawaver** [comment]
My 3rd try for the update was finally successful.
